Doc/library/ipaddress.rst

```diff
@@ -219,9 +219,14 @@ write code that handles both IP versions correctly.  Address objects are
       ``"::abc:7:def"``.
```

Comment:
(This would replace item 4 just below.)

```suggestion
       ``"::abc:7:def"``.

       Optionally, the string may also have a scope zone ID, expressed with a suffix ``%<scope_id>``. If present, the scope ID must be non-empty, and may not contain ``%``. See :RFC:`4007` for details. For example, ``fe80::1234%1`` might identify address ``fe80::1234`` on the first link of the node.  
```

Reply:
Fixed.